setup/so-setup

```diff
@@ -130,6 +130,7 @@ if [[ -f automation/$automation && $(basename $automation) == $automation ]]; th
 	if [[ ! $is_iso ]]; then
 		echo "Installing sshpass for automated testing." >> $setup_log 2>&1
 		if [ "$OS" == ubuntu ]; then
+			retry 50 10 "apt-get update" >> "$setup_log" 2>&1 || exit 1
```

Comment:
Are there actual scenarios where the initial "apt update" did not get invoked at the beginning of setup, on line 973? I'm trying to understand if this line and the line 1965 are needed.

Reply:
I originally added the initial `apt-get update` on 973 to the if statement several lines below, and missed removing these when I moved it out. They aren't necessary anymore, you're correct.